runtime/Cpp/runtime/src/antlr4-common.h

```diff
@@ -76,7 +77,7 @@
   class ANTLR4CPP_PUBLIC std::exception; // Needed for VS 2015.
 #endif
 
-#elif __APPLE__
+#elif defined(__APPLE__)
```

Comment:
Here's the message:

`antlr4/runtime/Cpp/runtime/src/antlr4-common.h:80:7: error: '__APPLE__' is not
      defined, evaluates to 0 [-Werror,-Wundef]
#elif __APPLE__
`